x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/api_key.ts

```diff
@@ -263,11 +263,11 @@ export default function apiKeyBackfillTests({ getService }: FtrProviderContext)
         }
       });
 
-      // invoke the invalidate task
-      await runInvalidateTask();
-
       // pending API key should now be deleted because backfill is done
       await retry.try(async () => {
```

Comment:
Seems like sometimes the last API key is not invalidated so moving this into the retry loop to retry to invalidate task as well.